vuepress/docs/next/tutorials/getting-started/openshift-install-by-operator.md

```diff
@@ -140,6 +140,14 @@ Now let's create a new application similar to Scenario 3, but with the self-sign
 
 Once the deployment is complete, you can confirm that all routes use HTTPS with the self-signed certificate. You may still see security warnings in the browser.
 
+#### Note: Multiple Certificate Chains
```

Comment:
1. A certificate chain by definition involves multiple certificates so we can simplify the title to Certificate Chain.
2. Maybe we should either move this note up to step 4, or connect to it here as a foot note. Otherwise they may do the whole thing and then only see this note at the bottom after it fails.